airflow/providers/openlineage/utils/sql.py

```diff
@@ -0,0 +1,194 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why do we have the `utils` folder and didn't add `sql` to the provider root? If we need a folder, would there be more specific names?